app/models/settings/smtp.rb

```diff
@@ -2,7 +2,7 @@ module Settings
   class SMTP < Base
     self.table_name = :settings_smtp
 
-    OPTIONS = %i[address port authentication user_name password domain].freeze
+    OPTIONS = %i[user_name password address authentication domain port from_email_address reply_to_email_address].freeze
```

Comment:
It seems this whole constant is only used within `app/views/admin/settings/forms/_smtp.html.erb`. We didn't even really need it, because

```ruby
Settings::SMTP::OPTIONS == Settings::SMTP.settings.keys
#=> true
```

Now that you want to apply a specific ordering, I'm wondering if we should use an `smpt_options` helper or something instead so we don't keep something that's a view concern in the model layer.

Reply:
@citizen428 this makes a lot of sense - thank you. I've updated it accordingly. Ready for review ✅